source/en/mongoid/docs/documents.haml

```diff
@@ -185,16 +185,53 @@
   :coderay
     #!ruby
 
-    # Get the value of the first name field.
-    person.first_name
+    # Get the value of the first_name field (raw data)
     person[:first_name]
     person.read_attribute(:first_name)
 
-    # Set the value for the first name field.
-    person.first_name = "Jean"
+    # Get the values of the first_name field (via getter, if any defined in the model as 'def first_name; ...; end')
+    person.first_name
```

Comment:
`person.first_name` should be before the ones that get the raw value. As this should be the most used.
also I would keep it simple, maybe something like:
`# Get the values of the first_name field (via getter)`